File: Triplet Comparer/Triplet Comparer.py
#!/bin/python3

import math
import os
import random
import re
import sys

def compareTriplets(a, b):
    alice = 0
    bob = 0
    for x, y in zip(a, b): #Use a zip to "combine" the 2 arrays, easy to get values
        if(x > y):
            alice += 1
        elif(y > x):
            bob += 1
        # Do nothing if the same
    results = [alice, bob]
    return results

if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    a = list(map(int, input().rstrip().split()))

    b = list(map(int, input().rstrip().split()))

    result = compareTriplets(a, b)

    fptr.write(' '.join(map(str, result)))
    fptr.write('\n')

    fptr.close()

File: Triplet Comparer/Triplet Comparer.rs
use std::env;
use std::fs::File;
use std::io::{self, BufRead, Write};

fn compareTriplets(a: &[i32], b: &[i32]) -> Vec<i32> {
    let mut alice = 0;
    let mut bob = 0;
    for (x, y) in a.iter().zip(b.iter()) {
        if x > y{
            alice += 1;
        }
        else if y > x{
            bob += 1;
        }
    }
    let result = vec![alice, bob];
    return result
}

fn main() {
    let stdin = io::stdin();
    let mut stdin_iterator = stdin.lock().lines();

    let mut fptr = File::create(env::var("OUTPUT_PATH").unwrap()).unwrap();

    let a: Vec<i32> = stdin_iterator.next().unwrap().unwrap()
        .trim_end()
        .split(' ')
        .map(|s| s.to_string().parse::<i32>().unwrap())
        .collect();

    let b: Vec<i32> = stdin_iterator.next().unwrap().unwrap()
        .trim_end()
        .split(' ')
        .map(|s| s.to_string().parse::<i32>().unwrap())
        .collect();

    let result = compareTriplets(&a, &b);

    for i in 0..result.len() {
        write!(&mut fptr, "{}", result[i]).ok();

        if i != result.len() - 1 {
            write!(&mut fptr, " ").ok();
        }
    }

    writeln!(&mut fptr).ok();
}
